integration/test/performance_tracker/transmitter/transmitter.go

```diff
@@ -0,0 +1,236 @@
+package transmitter
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/session"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/dynamodbiface"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+const METRIC_PERIOD= 300.0 // this const is in seconds , 5 mins
+
+type TransmitterAPI struct{
+	dynamoDbClient dynamodbiface.DynamoDBAPI
+	DataBaseName      string // this is the name of the table when test is run
```

Comment:
**Nitpick**: Would this use elsewhere `DataBaseName`? If not then suggesting to use `dataBaseName`